lib/github.go

```diff
@@ -189,7 +191,7 @@ func DetectAsset(userOS string, userArch string, releaseAssets []string) (string
 			}
 		}
 		if finalAsset == "" {
-			finalAsset, err = WarningPromptSelect("Could not automatically detect the release asset matching your OS/Arch. Please select it manually:", filteredReleaseAssets)
+			finalAsset, err = WarningPromptSelect("Could not automatically detect the release asset matching your OS/Arch. Please select it manually:", detectedFinalAssets)
```

Comment:
I think if filtering fails, I'd rather be conservative and show the user everything, so could you please revert this?

Reply:
Ok

But can we add some logic around interactive and non-I reactive terminals? I use stew in docker installations and “I will show you everything” is not working well in this scenario

Reply:
Yes it can be an option, but everybody will forget it on a first run. Just like -y is always forgotten in “apt install package“ in scripts

Reply:
For non-interactive installs, you should be installing from a `Stewfile.lock.json`.

```sh
stew install Stewfile.lock.json
```

Reply:
Eh. It will not work this way. I'm on Mac, my container is linux and maybe on different architecture